conwayste/src/client.rs

```diff
@@ -430,9 +436,9 @@ impl MainState {
 
         graphics::set_fullscreen(ctx, config.is_fullscreen() == true);
         vs.is_fullscreen = config.is_fullscreen() == true;
-*/
+        */
 
-        let viewport = viewport::Viewport::new(config.get_zoom_level(), universe_width_in_cells, universe_height_in_cells);
+        let viewport = viewport::Viewport::new(config.get().gameplay.zoom, universe_width_in_cells, universe_height_in_cells);
```

Comment:
I don't know why it just clicked now but Rust's borrowing/mutability rules finally comfort me with this kind of usage. Didn't like the "openness" of it all but now I get it. :+1: